### PR TITLE
Add Codespace Latest

### PR DESCRIPTION
**Pull request by Crysicia**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/codespace.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'codespace' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://codespace.app/download/latest'
```
Comment (vitorgalvao) [inline Casks/codespace.rb:5]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md). `https://macupdater.net/cgi-bin/check_urls/check_url_filename.cgi?url=https://codespace.app/download/latest` if you can’t find anything else.

File: Casks/codespace.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'codespace' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/codespace.rb:2]: Add a `version` and `sha256`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/codespace.rb (hunk 1 above)
Comment (Crysicia) [reply Casks/codespace.rb:5]: Done!
I've used the [releases](https://codespace.app/releases) page, is that ok ?

File: Casks/codespace.rb (hunk 2 above)
Comment (Crysicia) [reply Casks/codespace.rb:2]: Unfortunately only the latest version of Codespace is available for download, I'm afraid I cannot give a version number.

File: Casks/codespace.rb (hunk 1 above)
Comment (vitorgalvao) [reply Casks/codespace.rb:5]: Use the one I gave. It’s better because it only has one number.

File: Casks/codespace.rb (hunk 2 above)
Comment (vitorgalvao) [reply Casks/codespace.rb:2]: You can. Make it `1.1.2`. There’s an appcast, so the `url` doesn’t need to have a version in it (though that would be preferable).

**Review by vitorgalvao — APPROVED** [review]
Thank you for the fast turnaround. Welcome!

**Comment by Crysicia** [comment]
Thanks a lot for your help, first submission here :)

**Pull request merged**